source/agora/consensus/data/Block.d

```diff
@@ -120,6 +136,72 @@ unittest
     }();
 }
 
+/*******************************************************************************
+
+    ValidatorSigs must be signed with a valid freeze utxo of the Validator
+    in the blockheader.
+
+*******************************************************************************/
+
+public struct ValidatorSigs
```

Comment:
The structure name, ValidatorSigs is confusing because it make developers to think that the structure has multiple signatures. You should use the name like ValidatorSig.

Reply:
applied the suggested.